### PR TITLE
Style ODataReact component

### LINKED ISSUE #4: Style OData component
**Description**

As inlined in https://github.com/kyma-project/console/issues/684, we need to make our styles consistent across specification components. This issue is for OData component.

**Reasons**

To have consistent style across application.

See also https://github.com/kyma-project/console/issues/684

### PR DESCRIPTION
**Pull request by aerfio**
**Description**

Changes proposed in this pull request:

- update styling of odata component
- fix bug with expand/collapse all

**Related issue(s)**

Resolves #4 
Fixes https://github.com/kyma-project/console/issues/672

### CODE REVIEW

File: tsconfig.base.json [hunk 1]
```diff
@@ -4,7 +4,7 @@
   "compilerOptions": {
     "target": "es5",
     "module": "commonjs",
-    "lib": ["dom", "es2017"],
+    "lib": ["dom", "es2017", "esnext"],
```
Comment (magicmatatjahu) [inline tsconfig.base.json]: I think that `"es2017"` is unnecessary 😄

File: packages/odata-react/src/components/styled/styled.ts [hunk 2]
```diff
@@ -111,53 +161,40 @@ export const NotFoundMessage = styled.p`
...
+NotFoundMessage.displayName = "NotFoundMessage";
 
 Icon.defaultProps = {
   size: "l",
   glyph: "feeder-arrow",
 };
 
-export const CollapseButton = styled(Button)`
+export const CollapseButton = styled(Button).attrs(() => ({
+  className: "CollapseButton",
```
Comment (magicmatatjahu) [inline packages/odata-react/src/components/styled/styled.ts]: Please use kebabcasing class names everywhere.

File: tsconfig.base.json (hunk 1 above)
Comment (aerfio) [reply tsconfig.base.json]: it is neccesary for `flatMap`

File: packages/odata-react/src/store/index.tsx [hunk 3]
```diff
@@ -0,0 +1,21 @@
+import { useState } from "react";
+import createUseContext from "constate";
+
+interface StoreShape {
+  [key: string]: boolean;
+}
+
+const useCollapseStateRaw = () => {
+  const [state, setState] = useState<StoreShape>({});
+  const updatePart = (arg: StoreShape) => setState({ ...state, ...arg });
```
Comment (magicmatatjahu) [inline packages/odata-react/src/store/index.tsx]: Please change name of this function and also for `updateWholeState`. For my naming like `updateState` etc by related to specific part of state is very unreadable to 3rd person, so `updatePartOfCollapseState` would be better.

**Pull request merged**